graphics/squish/Portfile

```diff
@@ -37,3 +49,14 @@ array set merger_configure_args {
     ppc64   "BUILD_SQUISH_WITH_ALTIVEC=ON   BUILD_SQUISH_WITH_SSE2=OFF"
     x86_64  "BUILD_SQUISH_WITH_ALTIVEC=OFF  BUILD_SQUISH_WITH_SSE2=ON"
 }
+
+post-destroot {
+     foreach re [list \
+         "s|@PREFIX@|${prefix}|" \
+         "s|@LIB_PATH@|lib|" \
+         "s/\(Version: \).*/\\1${version}/" \
+     ] {
+         reinplace -E $re \
+             ${destroot}${prefix}/share/pkgconfig/lib${name}.pc
```

Comment:
So the build system doesn't set up the pc file correctly? Ordinarily I'd like to see a comment with a bug report URL accompany such changes, since this is not a MacPorts-specific problem. Unfortunately I see the project is hosted on sourceforge and doesn't have an issue tracker set up. Still, the [author](https://sourceforge.net/u/roettger/profile/) seems to be active, having committed something just yesterday, so some effort should be made to inform them of this problem.

Reply:
> So the build system doesn't set up the pc file correctly?

CMake can be made to do so, but it involves adding more code than the few lines that I did.

> Still, the author seems to be active, having committed something just yesterday, so some effort should be made to inform them of this problem.

Perhaps, but if you look at the page for [libsquish](https://sourceforge.net/projects/libsquish/files/), the latest source code tarball was released 2017-03-14.

Reply:
> > So the build system doesn't set up the pc file correctly?
> 
> CMake can be made to do so, but it involves adding more code than the few lines that I did.

I moved the setup of the pc file into the `CMakeLists.txt` file, since I was extracting the reinplace to a patchfile anyway, as per your other conversation.

> > Still, the author seems to be active, having committed something just yesterday, so some effort should be made to inform them of this problem.
> 
> Perhaps, but if you look at the page for [libsquish](https://sourceforge.net/projects/libsquish/files/), the latest source code tarball was released 2017-03-14.

A couple other indications of how inactive this library is, even if the author remains active. At the top of the `CMakLists.txt` file:

```
CMAKE_MINIMUM_REQUIRED(VERSION 2.8.3)
```

Also, if you look at the remaining reinplace in my `post-destroot` section, I still have to replace the version number in the `.pc` file, because the `libsquish.pc.in` file has this:

```
Version: 1.14
```

even though the source tarball is for 1.15.